.github/scripts/apply_review.py

```
@@ -173,7 +173,7 @@ def _write_file(self, path: str, content: str) -> str:
     def _bash(self, cmd: str) -> str:
         env = {k: v for k, v in os.environ.items() if k not in _STRIP_KEYS}
         r = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)